dev-tools/validation/check-authz-route-matrix.py

```
@@ -44,81 +62,2237 @@
+    for route in routes:
+        if not isinstance(route, dict):
+            continue
+        key = (route.get("service"), route.get("route"))
+        if key not in EXPLICIT_NO_JWT_ROUTES:
+            continue
+        label = f"{route.get('service')} {route.get('route')}"
+        if route.get("accepted_token_profiles") != []:
+            errors.append(f"{label} must explicitly declare accepted_token_profiles=[]")
+        for field in ("token_type", "token_issuer", "token_audience"):
+            if route.get(field) != "none":
+                errors.append(f"{label} must explicitly declare {field}=none")
```

Ruff (0.16.0)

[warning] Use `list.extend` to create a transformed list

(PERF401)